infrastructure/conf/nginx-conf-local/default

```diff
@@ -82,6 +82,11 @@ server {
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME /home/site/wwwroot/api/public/index.php;
     }
+    location = /api/support/tickets {
```

Comment:
Can we just capture anything starting with `/api`? Might avoid future issues?

Reply:
@tristan-orourke please see [Update nginx location block to capture all api endpoints](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5040/commits/a915b00779d902e802ada21ea9a5bee400a38e3f).